corehq/apps/app_manager/suite_xml/post_process/workflow.py

```diff
@@ -461,7 +461,7 @@ def get_case_type_created_by_form(form, target_module):
                 return form.get_module().case_type
             else:
                 return reg_action.case_type
-        elif form.form_type == 'advanced_form':
+        elif form.form_type == 'advanced_form' or form.form_type == "shadow_form":
```

Comment:
They can't have their "own" registration actions, but they do inherit them from their source form, so I think this is still needed.